### PR TITLE
Configure fluent-testing for publishing

### PR DESCRIPTION
**Pull request by nordzilla**
- Adds dual licence files
- Adds readme
- Adds appropriate metadata to Cargo.toml

### CODE REVIEW

**Review by zbraniecki — APPROVED** [review]

File: fluent-testing/Cargo.toml [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+"""
+version = "0.0.2"
+authors = [
+    "Zibi Braniecki <zibi@braniecki.net>",
+    "Erik Nordin <enordin@mozilla.com>"
+]
 edition = "2018"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+homepage = "http://www.projectfluent.org"
```
Comment (eemeli) [inline fluent-testing/Cargo.toml:12]:
```suggestion
homepage = "https://www.projectfluent.org"
```

File: fluent-testing/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+`fluent-testing` is a collection of mock scenarios for testing fluent-rs components.
+
+Project Fluent keeps simple things simple and makes complex things possible.
+The syntax used for describing translations is easy to read and understand.  At
+the same time it allows, when necessary, to represent complex concepts from
+natural languages like gender, plurals, conjugations, and others.
+
+[Documentation][]
+
+[Project Fluent]: http://projectfluent.org
```
Comment (eemeli) [inline fluent-testing/README.md:12]:
```suggestion
[Project Fluent]: https://projectfluent.org
```

File: fluent-testing/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+FTL is designed to be simple to read, but at the same time allows to represent
+complex concepts from natural languages like gender, plurals, conjugations, and
+others.
+
+    hello-user = Hello, { $username }!
+
+[Read the Fluent Syntax Guide][] in order to learn more about the syntax.  If
+you're a tool author you may be interested in the formal [EBNF grammar][].
+
+[Read the Fluent Syntax Guide]: http://projectfluent.org/fluent/guide/
```
Comment (eemeli) [inline fluent-testing/README.md:52]:
```suggestion
[Read the Fluent Syntax Guide]: https://projectfluent.org/fluent/guide/
```

**Review by eemeli — APPROVED** [review]
Nitpicky link fixes that'll get auto-redirected in any case.

**Comment by zbraniecki** [comment]
@nordzilla can you unbitrot and merge?

**Comment by zbraniecki** [comment]
@nordzilla can you resolve the conflict and merge?

**Comment by gregtatum** [comment]
I rebased and merged.

**Pull request merged**